docs/platforms/javascript/common/session-replay/troubleshooting.mdx

```diff
@@ -46,6 +46,27 @@ Due to [browser limitations](https://developer.mozilla.org/en-US/docs/Web/SVG/El
 </Alert>
 </Expandable>
 
+<Expandable title="Why am I seeing too many or too few rage clicks?" permalink>
+
+There might be fewer rage clicks than you expect if the user stopped waiting for the site to respond before the 7 second threshold and instead chose to to something else. This is why the rage click issues that you *do* see are so valuable, because the user that clicked at least 3 times and continued waiting at least 7 seconds for the site to respond is likely very frustrated.
```

Comment:
```suggestion
There might be fewer rage clicks than you expect if the user stopped waiting for the site to respond before the 7 second threshold. This is why the rage click issues that you *do* see are so valuable, because the user that clicked at least 3 times and continued waiting at least 7 seconds for the site to respond is likely very frustrated.
```